docs/cpp/cpp-ide.md

```diff
@@ -1,42 +1,42 @@
+DateApproved: 1/2/2024
 MetaDescription: How to edit and navigate C++ source files in Visual Studio Code.
 ---
-# Edit C++ in Visual Studio Code
+# Editing and Navigating
 
-This topic provides a quick overview of general C/C++ editor features, as well as some that are specific to C/C++. For more information about editing in Visual Studio Code, see [Basic Editing](/docs/editor/codebasics.md) and [Code Navigation](/docs/editor/editingevolved.md).
+This article provides an overview of code editing and navigating features specific to the C/C++ Extension. For more information about general editing and navigating in Visual Studio Code, see [Basic Editing](/docs/editor/codebasics.md) and [Code Navigation](/docs/editor/editingevolved.md).
 
-The C/C++ extension supports [Remote Development](/docs/remote/remote-overview.md).
+## Editing Set Up
 
-## Editing features
+To provide the best editing experience, the C++ extension needs to know where it can find each header file referenced in your code. By default, the extension searches the current source directory, its subdirectories, and some platform-specific locations. If a referenced header file can't be found, a red squiggle is displayed underneath the #include directive.
 
-The C/C++ extension for VS Code has many features that help you write code, understand it, and navigate around in your source files. To provide the best experience, the extension needs to know where it can find each header file referenced in your code. By default, the extension searches the current source directory, its sub-directories, and some platform-specific locations. If a referenced header file can't be found, VS Code displays a green squiggle underneath each #include directive that references it.
+To specify additional include directories, select an #include that has no reference. Next, select the light bulb that appears and choose `Edit "includePath" setting`, which opens the C/C++ Extension's Configurations interface. Under the Include Path section, you can specify the paths for any additional include directories.
 
-To specify additional include directories to be searched, place your cursor over any #include directive that displays a green squiggle, then click the lightbulb action when it appears. This opens the file `c_cpp_properties.json` for editing; here you can specify additional include directories for each platform configuration individually by adding more directories to the 'browse.path' property.
+![Process of adding a new header to the include path](images\intellisense\AddingIncludePathToConfig.gif)
 
 ## List members
 
-When you type a member access symbol (`.` or `->`) the editor will display a list of members. As you type additional letters, the list is filtered in real time:
+When you type a member access symbol (`.` or `->`) the editor displays a list of members. As you type more letters, the list is filtered in real time:
 
 ![List members](images/cpp/list-members-cpp.png)
 
 ### Code formatting
```

Comment:
It has been a common issue in the past that users are unaware that formatting will not occur if IntelliSense is disabled (https://github.com/microsoft/vscode-cpptools/issues/7083, https://github.com/microsoft/vscode-cpptools/issues/7339) . Maybe make a small note of it here with an asterisk (*)?

Reply:
I vote that we don't add a note for that case since we don't want to inadvertently suggest that disabling IntelliSense is a good idea.

Reply:
I think it would be better to approach it from the opposite side and highlight the positives of having IntelliSense -  add a question to our FAQ, like "My code formatting is not working. How can I fix it? " and then state they should double check that IntelliSense is enabled, which is required for code formatting.

Reply:
I would also vote not to have that in the FAQ